### DISCUSSION TITLE
[Suggestion] Add more configs

### DISCUSSION THREAD

**Discussion by XtraCube** [dimaguy/ImpostorHQ#12] [Ideas]
Add some more iHQ configs, like togglable chat commands. I use iHQ and my own plugin and I had to rebuild the iHQ source to remove it's chat commands so it didn't interfere with mine.

**Comment by dimaguy** [comment]
Most of the commands we have were moved over to separate plugins, please list the commands that you need to toggle as the ones that remain should be only the core ones
